### PR TITLE
New GUI pages, pages split out, some cleanup

### PR DESCRIPTION
**Pull request by Jaffacakelover**
Created pages for the other GUIs. Split "Starting a game" and "disk swapping" out into their own pages. Added new images. Added a few cross-references. General cleanup.

### CODE REVIEW

File: docs/image/retroarch/xmb/thumbnails1.png
Comment (fpscan) [inline docs/image/retroarch/xmb/thumbnails1.png:1]: Is it possible to have different thumbnails?

**Review by fpscan — CHANGES_REQUESTED** [review]
It is best to have non-nintendo stuff.

**Comment by Jaffacakelover** [comment]
Understandable, yes. Amending the pics now: If anyone can name this filter (or grab me a new AltStore screenshot) that'd help out.
screenshot5

**Comment by fpscan** [comment]
> Understandable, yes. Amending the pics now: If anyone can name this filter (or grab me a new AltStore screenshot) that'd help out. screenshot5
@hizzlekizzle any wild guesses?

**Comment by Jaffacakelover** [comment]
Found it: `Slang > Bezel > Mega Bezel`. I'll use `Presets > MBZ__2__ADV-GLASS` for my shot.

**Comment by Jaffacakelover** [comment]
That should do it, unless you want me to delve into all the per-Core images and Shader examples.

**Pull request merged**